go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/gardener/aws-lb-readvertiser
+
+go 1.14
```

Comment:
```suggestion
go 1.15
```

Can you also update this in the `Dockerfile`?

Reply:
Do you mean the image in the pipeline_defintions?

Reply:
Ah, this repository is still setup in a different/old way. Then I suggest to just update the alpine version here: https://github.com/gardener/aws-lb-readvertiser/blob/master/Dockerfile#L5